vowpalwabbit/parse_regressor.cc

```diff
@@ -18,6 +18,8 @@ using namespace std;
 #include <math.h>
 #include <algorithm>
 #include <stdarg.h>
+#include <numeric>
+#include <random>
```

Comment:
yes..I will switch to rand48.h and add the box-muller function to get normal dist. instead of <random>